send_message.py

```diff
@@ -4,22 +4,18 @@
 import smtplib
 import sys
 import os
- 
-CARRIERS = {
-    "att": "@mms.att.net",
-    "tmobile": "@tmomail.net",
-    "verizon": "@vtex.com",
-    "sprint": "@page.nextel.com"
-}
- 
-EMAIL = ""
-PASSWORD = ""
+import security as vault
 
-RECIPIENTS = []
+import sys
+sys.path.append('/')
```

Comment:
We should remove this as it isn't needed :D

Reply:
Done! :D